README.md

```diff
@@ -11,14 +11,16 @@ for any bug bounty. We advise you to be careful and experiment on the network at
 
 ## Cmd usage
 
-The latest release version is compatible with the greenfield. If the user need to use the command tool on the test net of greenfield, he will need to switch to the latest release version.
+Please notice that the latest release version is compatible with the greenfield system of test net. 
+If the user need to use the command tool on the test net of greenfield, he will need to switch to the latest release version.
+To obtain the latest version of the command tool, please visit the following URL: https://github.com/bnb-chain/greenfield-cmd/releases.
+
 ### installation
 
 ```
 git clone https://github.com/bnb-chain/greenfield-cmd.git
 cd greenfield-cmd
-# To obtain the latest version of the command tool, please visit the following URL: https://github.com/bnb-chain/greenfield-cmd/releases.
-git checkout -b v0.0.8
```

Comment:
Add a line: # Find the latest release here: https://github.com/bnb-chain/greenfield-cmd/releases

Reply:
added